README.md

```diff
@@ -33,9 +33,11 @@ The Node-Red-Contrib-Face-Vectorization requires [Node-RED](https://nodered.org)
 > This is a brief description of each node.
 
 - _Good Calculate Cosine_: It calculate cosine similarity between two vectors and return it as an array.
-- _Good Face Vectorization_: It converts facial photos to vectors using AI as an array.
+- _Good Face Vectorization_: It converts facial photos to vectors using FaceNet as an array.
 - _Good ChromaDB_: It connects and operates with ChromaDB server.
-
+- _Good Face Detection_ : It detects faces in the photo using yolov8.
+- _Good Object Detection_ : It detects objects in the photo using yolov8.
+- Good Webcam Object Detection\_ : It detects objects in the webcam.
```

Comment:
There seems to be a typo 'Good Webcam Object Detection\\'

Reply:
fixed thanks for review